Dockerfile

```diff
@@ -1,5 +1,8 @@
+FROM quay.io/costoolkit/releases-green:luet-toolchain-0.21.2 AS luet
```

Comment:
Why using `releases-green` instead of `releases-teal`?

Reply:
We'll omit the luet use once https://github.com/harvester/harvester/issues/2565 is done.
Here is to try to bring back what worked before to minimize risks. (luet is used in build time only).